examples/burial-form/MilitaryServiceHistory.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from 'react';
+import {
+  DateField,
+  Page,
+  TextField,
+  DebuggerView
+} from '@department-of-veterans-affairs/va-forms-system-core';
+import { useFormikContext } from 'formik';
+
+export default function MilitaryServiceHistory(props) {
+  useEffect(() => {
+    window.scrollTo({
+      top: 0,
+      left: 0,
+      behavior: "smooth"
+    });
+  });
+
+  const state = useFormikContext();
+
+  const isBeforeStartDate = (value) => {
+    if (state.values.toursOfDuty[0].dateRange.from >= value && value.length === 10) {
```

Comment:
Yeah basically, the chain validation function allows for a props.validate function to be passed. Also the date field produces a date string that can be compared against another date string.  The length part is there because I didn't want it to show red before the user finished entering a date.